src/test/java/HomeWork21.java

```diff
@@ -0,0 +1,16 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class HomeWork21 extends BaseTest{
+    @Test
+    public void renamePlaylist(){
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        loginPage.logIn();
+        homePage.doubleClickPlaylist();
+        homePage.enterNewPlaylistName();
+        Assert.assertTrue(homePage.doesPlaylistExists());
+    }
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/a8907b4f-7e24-45bb-af95-1e6a9ff87c01)